Scripts/Services/Malas/Sphynx/SphynxFortune.cs

```diff
@@ -218,37 +213,41 @@ public static void StartTimer()
             m_Timer = Timer.DelayCall(TimeSpan.FromMinutes(1), TimeSpan.FromMinutes(1), new TimerCallback(DefragTables));
             m_Timer.Start();
         }
-		
-		public SphynxFortune(Serial serial) : base(serial)
-		{
-		}
-		
-		public override void Serialize(GenericWriter writer)
-		{
-			base.Serialize(writer);
-			writer.Write((int)0);
 
-            writer.Write(Fountains.Count);
+        public static void OnSave(WorldSaveEventArgs e)
+        {
+            Persistence.Serialize(
+                FilePath,
+                writer =>
+                {
+                    writer.Write((int)0);
 
-            foreach (var pair in Fountains)
-            {
-                writer.Write(pair.Mobile);
-                writer.Write(pair.Date);
-                writer.Write(pair.Type.ToString());
-                writer.Write(pair.TypeValue.ToString());
-                writer.Write(pair.Value);
-            }
+                    writer.Write(Fountains.Count);
+
+                    Fountains.ForEach(s =>
+                    {
+                        writer.Write(s.Mobile);
+                        writer.Write(s.Date);
+                        writer.Write(s.Type.ToString());
+                        writer.Write(s.TypeValue.ToString());
+                        writer.Write(s.Value);
+                    });
+                });
         }
 
-        public override void Deserialize(GenericReader reader)
+        public static void OnLoad()
         {
-            base.Deserialize(reader);
-            int version = reader.ReadInt();
+            Persistence.Deserialize(
+                FilePath,
+                reader =>
+                {
+                    int version = reader.ReadInt();
 
-            for (int i = reader.ReadInt(); i > 0; i--)
-            {
-                Fountains.Add(new SphynxFortuneArray { Mobile = reader.ReadMobile(), Date = reader.ReadDateTime(), Type = (EnumType)Enum.Parse(typeof(EnumType), reader.ReadString()), TypeValue = (EnumTypeValue)Enum.Parse(typeof(EnumTypeValue), reader.ReadString()), Value = reader.ReadInt() });
-            }
+                    for (int i = reader.ReadInt(); i > 0; i--)
+                    {
```

Comment:
Mobile = reader.ReadMobile()

If this mobile is null, will this cause any adverse effects?  players that are deleted prior to server restarted will be null upon server start.